client/data/reader/recommendations/use-recommend-list-mutation.ts

```diff
@@ -0,0 +1,45 @@
+import { translate } from 'i18n-calypso';
+import { useDispatch } from 'calypso/state';
+import { Feed } from 'calypso/state/data-layer/wpcom/read/feed/types';
+import {
+	addRecommendedBlogsSite,
+	removeRecommendedBlogsSite,
+} from 'calypso/state/reader/lists/actions';
+import { useRecommendedList } from './use-recommend-list';
+
+type FeedId = Feed[ 'feed_ID' ];
+
+//TODO: Refactor to use react-query
+export const useRecommendedListMutation = ( owner: string ) => {
```

Comment:
im curious if we need this new hook, or if we can use the existing https://github.com/Automattic/wp-calypso/blob/trunk/client/landing/subscriptions/hooks/use-recommended-site.ts#L104 

That should give tell us whether the site is currently recommended, and give us a toggle function to add or remove. If that works, we may not need these 2 new hooks as it seems to do both?